numba/core/types/functions.py

```diff
@@ -202,21 +356,63 @@ def get_impl_key(self, sig):
 
     def get_call_type(self, context, args, kws):
         template = self.template(context)
-        e = None
+        literal_e = None
+        nonliteral_e = None
+        _DEBUG = False
+
+
         # Try with Literal
         try:
             out = template.apply(args, kws)
-        except Exception:
+        except Exception as exc:
```

Comment:
need to let `errors.ForceLiteralArg` propagate up

Reply:
done